quiz.js

```diff
@@ -37,7 +38,27 @@ db.dropDatabase();
 
 // your code begins here
 
+db.houses.update({
+  name: 'Stark'
+}, {
+  $push: {people: {
+    name: 'Ned Stark'
+  }}
+});
 
+db.houses.update({
+  name: 'Stark'
+}, {
+  $addToSet: { people: {$each: [{name: 'Arya Stark'}, {name: 'Sansa Stark'}]}}
+});
+
```

Comment:
Nice! I'd be interested to know why you chose to use $addToSet the second time instead of $push, but this code will totally work.